### ISSUE TITLE
BUG 01: Missing alt attributes on images (Accessibility Issue)

### ISSUE THREAD

**Issue by Iannjeru** [Iannjeru/Portfolio#1]
**Steps to Reproduce:**

1. Open gallery.html
2. Scroll to line 154 where the image tag is defined.
3. Observe that no alt attribute is provide

**Expected Behavior:**
Each image should include a meaningful alt description for accessibility.

**Actual Behavior:**
The image loads successfully but lacks descriptive text for assistive technologies.
**Recommendation:**
Add appropriate alt text to all <img> elements.
Severity: Medium
Evidence:

<img width="555" height="48" alt="Image" src="https://github.com/user-attachments/assets/1b469e94-684d-4c79-9499-7f4113eb431d" />
